docs/pypi.org/Terms-of-use.md

```diff
@@ -1,7 +1,7 @@
 ---
 source: https://pypi.org/policy/terms-of-use/
 ---
-# Terms of use
+# Terms of Use
```

Comment:
Note: if the case here changes, we'll need an update over here:
https://github.com/pypi/warehouse/blob/83b0c6de32c174208f3a65c061bd8fcc8983213c/warehouse/routes.py#L62 
(and associated test case) as mkdocs/rtd are case sensitive.

Reply:
Cool, I'll make a PR there as well!

Reply:
https://github.com/pypi/warehouse/pull/16210